SPECS/cockpit/cockpit.spec

```diff
@@ -0,0 +1,90 @@
+%define _unpackaged_files_terminate_build 0
+Summary:        A package to provide cockpit for mariner
+Name:           cockpit
+Version:        248
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/cockpit-project/cockpit
+Source0:        https://github.com/cockpit-project/cockpit/releases/download/248/cockpit-248.tar.xz
```

Comment:
>https://github.com/cockpit-project/cockpit/releases/download/248/cockpit-248.tar.xz [](http://example.com/codeflow?start=16&length=83)

Please use `%{name}` and `%{version}` macros for the source URL wherever possible (here and other specs). This makes updates easier. In this case you could do:

```
Source0:        [https://github.com/cockpit-project/%{name}/releases/download/%{version}/%{name}-%{version}.tar.xz](https://github.com/cockpit-project/%{name}/releases/download/%{version}/%{name}-%{version}.tar.xz)
```